.circleci/config.yml

```diff
@@ -608,7 +608,7 @@ jobs:
       <<: *default-build-environment
       OPENSSL_BIN: /usr/local/opt/openssl/bin/openssl
       SCCACHE_CACHE_SIZE: 450M
-      RUSTFLAGS: -D warnings -C target-cpu=penryn
+      RUSTFLAGS: -D warnings -C target-cpu=skylake
```

Comment:
This should stay as penryn, since Mac builds are client-only, `RUSTFLAGS` needs to be added to, e.g. `build-and-lint-debug` and the test runners.

Reply:
Oops, I misunderstood. Added `RUSTFLAGS` to `default-build-environment`